sentry_sdk/__init__.py

```diff
@@ -18,28 +18,32 @@
     "HttpTransport",
     "init",
     "integrations",
+    "trace",
     # From sentry_sdk.api
+    "add_breadcrumb",
     "capture_event",
-    "capture_message",
     "capture_exception",
-    "add_breadcrumb",
+    "capture_message",
     "configure_scope",
-    "push_scope",
+    "continue_trace",
     "flush",
+    "get_baggage",
+    "get_client",
+    "get_current_span",
+    "get_traceparent",
+    "is_initialized",
+    "isolation_scope",
```

Comment:
Why do we only have a getter for the `isolation_scope`?

Reply:
Here `isolation_scope` and `new_scope` reference the context managers defined in `scope.py` and imported in `api.py`.